client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -0,0 +1,208 @@
+				{ translate( 'Check Again' ) }
+			</Button>
+		);
+
+		return this.getSection( heading, message, explanation, 1, button );
+	}
+
+	getPrivacyMessage() {
+		const { translate } = this.props;
+		const { email, privacy } = this.state;
+
+		const heading = privacy
+			? translate( 'Privacy protection is disabled.' )
+			: translate( 'Disable privacy protection.' );
+		const message = privacy
+			? translate(
+					"We'll send an email to {{strong}}%(email)s{{/strong}} to start the transfer process. Don't recognize " +
+						"that address? Then you have privacy protection, and you'll need to turn it off before we start.",
+					{
+						args: { email },
+						components: { strong: <strong /> },
+					}
+				)
+			: translate(
+					"We'll send an email to {{strong}}%(email)s{{/strong}} to start the transfer process. After the transfer" +
+						'is complete you can enable privacy to hide your registration information again.',
+					{
+						args: { email },
+						components: { strong: <strong /> },
+					}
+				);
+		const explanation = translate(
+			"It's important that we be able to reach you, because the transfer involves few emails. Privacy protection is great, " +
+				'but means that your contact information is hidden. To continue with the transfer, turn privacy protection off for now ' +
+				'- you can re-enable it once the transfer is done.'
+		);
+
+		return this.getSection( heading, message, explanation, 2 );
+	}
+
+	getEppMessage() {
+		const { translate } = this.props;
+
+		const heading = translate( 'Get a domain authorization code.' );
+		const message = translate(
+			"You'll need this code to okay the transfer. " +
+				"We'll send you and email with a link to the place where you'll need to enter it."
+		);
+		const explanation = translate(
+			'A domain authorization code is a unique code linked only to your domain - kind of like a ' +
+				'password for your domain. Log in to your current domain provider to get one. ' +
+				"We call it a domain authorization code, but it's also called a secret code, auth code or EPP code."
+		);
+
+		return this.getSection( heading, message, explanation, 3 );
+	}
+
+	render() {
+		const { translate } = this.props;
+		const headerLabel = translate(
+			'Log into your current registrar to complete a few preliminary steps.'
+		);
+
+		return (
+			<div className="transfer-domain-step__precheck">
+				<SectionHeader label={ headerLabel } />
+				{ this.getStatusMessage() }
+				{ this.getPrivacyMessage() }
+				{ this.getEppMessage() }
+				<div className="transfer-domain-step__continue">
+					<div>
+						{ translate(
+							'Note: These changes can take up to 20 minutes to take effect.{{br/}}' +
+								'Need help? {{a}}Get in touch with one of our Happiness Engineers{{/a}}.',
+							{
+								components: {
+									a: <a href="#" />,
+									br: <br />,
+								},
+							}
+						) }
+					</div>
+					<Button disabled={ ! this.state.unlocked } onClick={ this.onClick }>
```

Comment:
Should we not check `this.state.privacy` also? I think it's pretty dependable? Or do we want to protect ourselves from false positives? Like transferring in domains from a "_Private_ Investigator"? ;P

Reply:
Yes, we do. Because `whois` might fail, and we can only `guess` the privacy. :)

Reply:
And often privacy is forwarding, which won't block the transfer. :)

Reply:
👍